core/unit_test/TestReducers_c.hpp

```diff
@@ -18,6 +18,12 @@
 
 namespace Test {
 TEST(TEST_CATEGORY, reducers_double) {
+#ifdef KOKKOS_ENABLE_OPENMPTARGET
```

Comment:
```suggestion
#ifdef KOKKOS_ENABLE_OPENMPTARGET  // FIXME_OPENMPTARGET
```
Any idea what the problem is?